### PR TITLE
Browser Recorder Blog

### PR DESCRIPTION
**Pull request by aryangupta701**

### CODE REVIEW

File: site/content/blog/2023-09-09-browser-extension/index.md
Comment (thc202) [inline site/content/blog/2023-09-09-browser-extension/index.md:3]: Needs update.

File: site/content/blog/2023-09-09-browser-extension/index.md
Comment (psiinon) [inline site/content/blog/2023-09-09-browser-extension/index.md:11]: OK to publish on Monday? Will give us more time to review and publish the client add-on.
I would expect it to get more visibility during a weekday as well 😁

**Comment by psiinon** [comment]
Definitely need to explain to people how they can try this out for themselves.
This means installing the client add-on from the Marketplace (we can provide the link), noting it only works with Firefox (for now) and that they have to approve for each site.
I think it will fail on Chrome as the latest version has not been approved yet? If so should we disable Chrome support in Zest until we can update it?

File: site/content/blog/2023-09-09-browser-extension/index.md
Comment (psiinon) [reply site/content/blog/2023-09-09-browser-extension/index.md:3]: "GSoC 2023 Browser Recorder" (or similar) might be a better blog title than "Browser Recorder - Exploring My Google Summer of Code 2023 Journey" - its better to keep them shorter (IMO). You can still use that title lower down..

File: site/content/blog/2023-09-09-browser-extension/images/zapgsoc.png
Comment (psiinon) [inline site/content/blog/2023-09-09-browser-extension/images/zapgsoc.png:1]: Trim some of the whitespace around the images?

File: site/content/blog/2023-09-09-browser-extension/index.md
Comment (psiinon) [inline site/content/blog/2023-09-09-browser-extension/index.md:59]: "Client add-on" and link to the relevant web page (we'll need to provide that in a bit)

File: site/content/blog/2023-09-09-browser-extension/index.md
Comment (psiinon) [inline site/content/blog/2023-09-09-browser-extension/index.md:66]: "Zest add-on" and link to https://www.zaproxy.org/docs/desktop/addons/zest/
I think it should always be "add-on" - lower case with a '-' might need to update a few other places..

**Review by psiinon — COMMENTED** [review]
Some of the titles are very "wordy" - I'd be tempted to make them shorter, but in the end thats up to you.

File: site/content/blog/2023-09-09-browser-extension/index.md
Comment (psiinon) [inline site/content/blog/2023-09-09-browser-extension/index.md:23]: This info would be better on a Team page.
You can add a page for yourself here: https://www.zaproxy.org/docs/team/ (after Core Team, before Extended). You can do that in another PR which would get merged first if you like 😁

File: site/content/blog/2023-09-09-browser-extension/index.md
Comment (aryangupta701) [reply site/content/blog/2023-09-09-browser-extension/index.md:11]: Yes sure

File: site/content/blog/2023-09-09-browser-extension/index.md
Comment (aryangupta701) [reply site/content/blog/2023-09-09-browser-extension/index.md:23]: okay

**Comment by aryangupta701** [comment]
ready for review.I think images are not looking good. Is there any way to modify these form md file directly? I wanted to align it in center and create a border too.

File: site/content/blog/2023-09-11-browser-recorder/index.md
Comment (thc202) [inline site/content/blog/2023-09-11-browser-recorder/index.md:5]: The link needs to be updated, `extension` → `recorder`.

File: site/content/blog/2023-09-11-browser-recorder/index.md
Comment (thc202) [inline site/content/blog/2023-09-11-browser-recorder/index.md:40]: For the first mention `[Client Side Integration add-on](/docs/desktop/addons/client-side-integration/)` and for the following ones `Client add-on` (capitalize client).

File: site/content/blog/2023-09-11-browser-recorder/index.md
Comment (thc202) [inline site/content/blog/2023-09-11-browser-recorder/index.md:54]: - See previous comment.
 - What does it mean "belongs to the ZAP extensions", the repository?

File: site/content/blog/2023-09-11-browser-recorder/index.md
Comment (thc202) [inline site/content/blog/2023-09-11-browser-recorder/index.md:56]: `and ZAP`

File: site/content/blog/2023-09-11-browser-recorder/index.md
Comment (thc202) [inline site/content/blog/2023-09-11-browser-recorder/index.md:57]: I think they should be linked when first mentioned (though does not hurt linking in several places).

File: site/content/blog/2023-09-11-browser-recorder/index.md
Comment (thc202) [inline site/content/blog/2023-09-11-browser-recorder/index.md:61]: `(/docs/desktop/addons/zest/)`

File: site/content/blog/2023-09-11-browser-recorder/index.md
Comment (thc202) [inline site/content/blog/2023-09-11-browser-recorder/index.md:83]: `Client`, `Zest`

File: site/content/blog/2023-09-11-browser-recorder/index.md
Comment (thc202) [inline site/content/blog/2023-09-11-browser-recorder/index.md:84]: `Chrome`

File: site/content/blog/2023-09-11-browser-recorder/index.md
Comment (thc202) [inline site/content/blog/2023-09-11-browser-recorder/index.md:87]: `Zest Record Button`

File: site/content/blog/2023-09-11-browser-recorder/index.md
Comment (thc202) [inline site/content/blog/2023-09-11-browser-recorder/index.md:91]: `Record Zest Script Dialog`

File: site/content/blog/2023-09-11-browser-recorder/index.md
Comment (thc202) [inline site/content/blog/2023-09-11-browser-recorder/index.md:93]: - `on Start Recording`.
- `you have entered`?

File: site/content/blog/2023-09-11-browser-recorder/index.md
Comment (thc202) [inline site/content/blog/2023-09-11-browser-recorder/index.md:94]: `see a`

File: site/content/blog/2023-09-11-browser-recorder/index.md
Comment (thc202) [inline site/content/blog/2023-09-11-browser-recorder/index.md:96]: - `Stand Alone`
- Worth noting that the default type selected is Session Management not Stand Alone, the screenshot shown in the blog post is also showing the Session Management type selected.

File: site/content/blog/2023-09-11-browser-recorder/index.md
Comment (thc202) [inline site/content/blog/2023-09-11-browser-recorder/index.md:100]: - Chrome/Firefox
- Might be a good idea to suggest the use of launched browsers, which will already have the extension.

File: site/content/blog/2023-09-11-browser-recorder/index.md
Comment (thc202) [inline site/content/blog/2023-09-11-browser-recorder/index.md:109]: `You do not`?

File: site/content/blog/2023-09-11-browser-recorder/index.md
Comment (thc202) [inline site/content/blog/2023-09-11-browser-recorder/index.md:113]: `Zest`

File: site/content/blog/2023-09-11-browser-recorder/index.md
Comment (aryangupta701) [reply site/content/blog/2023-09-11-browser-recorder/index.md:96]: Oh yes thank you for pointing that out.

File: site/content/blog/2023-09-11-browser-recorder/index.md
Comment (aryangupta701) [reply site/content/blog/2023-09-11-browser-recorder/index.md:54]: yes the ZAP extensions but i think add-on already conveys that message

File: site/content/blog/2023-09-11-browser-recorder/index.md
Comment (aryangupta701) [reply site/content/blog/2023-09-11-browser-recorder/index.md:100]: didn't understand the second point can you please explain

Comment (thc202) [reply site/content/blog/2023-09-11-browser-recorder/index.md]: Just mention that the user can launch the browsers from ZAP (and link to quick start docs).

File: site/content/blog/2023-09-11-browser-recorder/index.md
Comment (thc202) [inline site/content/blog/2023-09-11-browser-recorder/index.md:40]: `add-on`

File: site/content/blog/2023-09-11-browser-recorder/index.md
Comment (thc202) [inline site/content/blog/2023-09-11-browser-recorder/index.md:57]: Remove the domain, for own links it's better to not include the domain so they point to the running server (e.g. testing locally).

File: site/content/blog/2023-09-11-browser-recorder/index.md
Comment (thc202) [inline site/content/blog/2023-09-11-browser-recorder/index.md:61]: `Client`

File: site/content/blog/2023-09-11-browser-recorder/index.md
Comment (aryangupta701) [reply site/content/blog/2023-09-11-browser-recorder/index.md]: isn't this resolved yet?

Comment (thc202) [reply site/content/blog/2023-09-11-browser-recorder/index.md]: Probably, I didn't check the latest change.

File: site/content/blog/2023-09-11-browser-recorder/index.md
Comment (thc202) [inline site/content/blog/2023-09-11-browser-recorder/index.md:100]: `… can launch browsers from ZAP using the [Quick Start add-on](…/#manual-explore), which will already have the extension installed.`

File: site/content/blog/2023-09-11-browser-recorder/index.md
Comment (thc202) [inline site/content/blog/2023-09-11-browser-recorder/index.md:42]: `extension to the Chrome Web Store`

File: site/content/blog/2023-09-11-browser-recorder/index.md
Comment (thc202) [inline site/content/blog/2023-09-11-browser-recorder/index.md:86]: `on providing full support for Firefox`

File: site/content/blog/2023-09-11-browser-recorder/index.md
Comment (psiinon) [inline site/content/blog/2023-09-11-browser-recorder/index.md:34]: "visionary" - thats probably stretching it a bit 😉 There are other browser recorders out there...

**Review by psiinon — APPROVED** [review]

**Review by thc202 — APPROVED** [review]

**Pull request merged**
